client/src/components/Home.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+function Home() {
+  return (
+    <div>
```

Comment:
we could use here self-closing tag for the div (because it doesn't have any children 'inside')